tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
@uchitsa add `set -e` after this line and you will see that your script actually fails (because you don't have the `fixtures/simple/input.md` file)

Reply:
@uchitsa check this comment please